src/object.d

```diff
@@ -3722,6 +3722,201 @@ template _arrayOp(Args...)
     alias _arrayOp = arrayOp!Args;
 }
 
+// switch template - replaces code in rt/switch_.d
+int __switch(T, caseLabels...)(T[] condition)
```

Comment:
It would be good to have a description here in comments of how this function is used by the compiler, especially since rt/switch_.d is going to be removed.

Reply:
Starting with copying/pasting the current documentation from rt/switch_.d

https://github.com/dlang/druntime/blob/970b5b2a3609873c1a4df377773a0dfd4524e5ba/src/rt/switch_.d#L18-L26

It looks like you just need to replace `table` with `caseLabels` and `ca` with `condition`.

Reply:
Right, but a good thing to note here, especially since it was confusing to most reviewers, is that the "sorted array" is sorted in an odd way (length first, then lexicographically).